lib/modules/apostrophe-pieces/index.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
   },
 
   beforeConstruct: function(self, options) {
-    self.contextual = options.contextual;
+    self.contextual = true ? (options.contextual !== false) : false;
```

Comment:
this doesn't make sense, the else clause can never happen. The thing you are testing is the constant: `true`